core/src/main/java/com/redhat/lightblue/client/LightblueClientConfiguration.java

```diff
@@ -6,6 +6,21 @@
 
 public class LightblueClientConfiguration {
 
+    public enum Compression {
+        NONE, LZF;
+
+        public static Compression parseCompression(String str) {
+            switch (str) {
+            case "NONE":
+                return NONE;
+            case "LZF":
+                return LZF;
+            default:
+                throw new IllegalArgumentException("Invalid compression " + str + ". Supported values are: NONE, LZF");
```

Comment:
Should NONE just be the default for backwards compatibility purposes?
Are there any legitimate reasons someone would not want to use compression? If not, then maybe that should just be the default?